plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -76,7 +76,6 @@ def sync_repo(self, repo, sync_conduit, config):
         :return: report of the details of the sync
         :rtype:  pulp.plugins.model.SyncReport
         """
-        repo = model.Repository.objects.get_repo_or_missing_resource(repo.id)
```

Comment:
The whole plugin system (conduits, importers, distributors, steps, etc.) needs to pass around the old-style Repository object so that non-converted plugins will work. It was just an oversight/mistake that one or two places in the steps happened to use the new-style Repository model, and that caused problems with compatibility across plugins. Those problems surfaced this week, hence the flurry of fixes.